spring-xd-messagebus-rabbit/src/main/java/org/springframework/xd/dirt/integration/rabbit/RabbitMessageBus.java

```diff
@@ -243,16 +246,19 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 				}
 			};
 
+	private static final ExpressionParser EXPRESSION_PARSER =
+			new SpelExpressionParser(new SpelParserConfiguration(true, true));
```

Comment:
This is existing code, but since you do the refactoring it would be better to optimize it a bit without `autoGrowNullReferences` and `autoGrowCollections`.